pulumi-orca/src/main/kotlin/com/pulumi/plugin/preconfigured/PulumiPlugin.kt

```diff
@@ -21,24 +21,12 @@ class PulumiPlugin(wrapper: PluginWrapper): Plugin(wrapper) {
 }
 
 @Extension
-class PulumiPreConfiguredStage(val pluginSdks: PluginSdks, val configuration: PluginConfig) : PreconfiguredJobConfigurationProvider {
```

Comment:
Unfortunately, I had to remove this because of https://github.com/pulumi/spinnaker-preconfigured-job-plugin/pull/9/files#r560643869. I do need to restore this, though.